integration_tests/pages/barcode/printCoversheets.ts

```diff
@@ -1,17 +1,26 @@
 /* eslint-disable import/no-cycle */
 import Page from '../page'
 import SelectEnvelopeSizePage from './selectEnvelopeSize'
+import FindRecipientByPrisonNumberPage from './findRecipientByPrisonNumber'
 
 export default class PrintCoversheetsPage extends Page {
   constructor() {
-    super('print-coversheets', { axeTest: false })
-    /*
-     axeTest (2nd constructor arg) must be false because axe relies on the browser tab being the active tab so that it can
-     interrogate the DOM. This page however auto-downloads the coversheet PDF as part of page load, so by the time axe runs
-     the active tab is not the page under test (even though the tab is still visible and the PDF has been downloaded rather than
-     opened in a new tab)
-     */
+    super('print-coversheets')
   }
 
+  downloadThePdfAgain = (): PrintCoversheetsPage => {
+    this.downloadLink().then(link => {
```

Comment:
This is one of the hacks I mentioned above that I've had to keep. The obvious thing to have done is:
```
this.downloadLink().click()
```
But cypress expects a `click` event to render a new page and so waits for a page load event from the browser ... which in this case never comes!
So we have to use cypress' `then` method to treat the element as a jquery element and use jquery's API to click it! 😢